### ISSUE TITLE
BitSubAccount.changeOwner Throw Error Invalid hex string

### ISSUE THREAD

**Issue by CalvinLeeC7E** [dotbitHQ/dotbit.js#47]
https://github.com/dotbitHQ/dotbit.js/blob/b560a15dd7f1a71f335c18bd5346ac36768f5497/src/signers/BitSigner.ts#L93

error: Invalid hex string

**Comment by web3max** [comment]
> https://github.com/dotbitHQ/dotbit.js/blob/b560a15dd7f1a71f335c18bd5346ac36768f5497/src/signers/BitSigner.ts#L93
> 
> error: Invalid hex string

Can you provide me with an example that can reproduce this problem?

**Comment by CalvinLeeC7E** [comment]
this api:

https://subaccount-api.did.id/v1/sub/account/edit

response is "From .bit: 123456"

```javascript
Buffer.from("From .bit: 123456", "hex") // error: Invalid hex string
```

**Comment by web3max** [comment]
> this api:
> 
> https://subaccount-api.did.id/v1/sub/account/edit
> 
> response is "From .bit: 123456"
> 
> ```js
> Buffer.from("From .bit: 123456", "hex") // error: Invalid hex string
> ```

We have just released 0.4.19 to fix this issue.
